conf/distro/lmp.conf

```diff
@@ -17,6 +17,8 @@ IMAGE_BOOT_FILES_remove_sota = " ${KERNEL_IMAGETYPE}"
 SOTA_CLIENT_PROV ?= ""
 ## Also produce OTA rootfs tarball for additional image support
 BUILD_OTA_TARBALL = "1"
+## Support Aktualizr with HSM by default for secure SOTA
+SOTA_CLIENT_FEATURES_append = " hsm"
```

Comment:
Does this need an additional `CORE_IMAGE_EXTRA_INSTALL += " softhsm "`?

Reply:
Ah NVM, it's below, sorry

Reply:
Yeah, it is part of the image via lmp-image-common.

With the changes in this pull request we don't need to set anything in local.conf anymore.